content/en/docs/contributing/testing.md

```diff
@@ -28,4 +28,39 @@ $ git format-patch HEAD~5
 $ ./support/scripts/checkpatch.pl ./000*
 ```
 
+You can also run the `checkpatch` on one or multiple files before committing.
+To do this, you can run the following command from the `root` directory of your repository clone:
```

Comment:
I would group all the information here, then give the example.
Something like:

```
You can also run the `checkpatch` on one or multiple files before committing by using the `-f` option.
Some of the errors can be solved automatically by using the `--fix-inplace` option:
```

Then give one example with both running the `checkpatch` and fixing the errors and state that you should be in the clone root directory.

Reply:
I think there is already stated above , so I would not mention it again.